api/pom.xml

```diff
@@ -50,9 +50,14 @@
         <!-- Required to generate CRD file -->
         <dependency>
             <groupId>io.fabric8</groupId>
-            <artifactId>crd-generator-apt</artifactId>
+            <artifactId>crd-generator-api-v2</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>io.sundr</groupId>
```

Comment:
Why do we need this dependency? From the fabric8 release notes it seemed like the old api was based on Sundrio, but not the new one

Reply:
We are using sundrio annotations in the kafka-access code (to create builder classes e.g. for KafkaAccess).
We didn't need to add the dependency previously as it was coming in from crd-generator-apt, however as the new crd-generator-api-v2 does not include it, we must now add it to continue using those annotations in the kafka-access code